-Wformat warning from `include/Rcpp/date_datetime/date_datetime.h:29`

I enabled some more warnings when compiling a package that uses Rcpp and got this warning which seems to be caused by Rcpp itself:
```
In file included from /usr/local/lib/R/site-library/Rcpp/include/Rcpp/date_datetime/date_datetime.h:29:
/usr/local/lib/R/site-library/Rcpp/include/Rcpp/date_datetime/Datetime.h:83:65: warning: format specifies type 'int' but the argument has type 'unsigned int' [-Wformat]
   83 |                 res = ::snprintf(txtsec, 63, "%s.%06d", txtiso, m_us);
      |                                                  ~~~~           ^~~~
      |                                                  %06u
```
It probably doesn't make any practical difference for the result, but it's nice if Rcpp itself doesn't trigger common compiler warnings.

--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2026-01-20  Dirk Eddelbuettel  <edd@debian.org>
+
+	* inst/include/Rcpp/date_datetime/Datetime.h (format): Correct a
+	format string reflecting 'unsigned int' rather than 'int'
+
 2026-01-12  Dirk Eddelbuettel  <edd@debian.org>
 
 	* DESCRIPTION (Version, Date): Roll micro version and date

--- a/inst/include/Rcpp/date_datetime/Datetime.h
+++ b/inst/include/Rcpp/date_datetime/Datetime.h
@@ -1,8 +1,7 @@
-// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
-//
+
 // Datetime.h: Rcpp R/C++ interface class library -- Datetime (POSIXct)
 //
-// Copyright (C) 2010 - 2016  Dirk Eddelbuettel and Romain Francois
+// Copyright (C) 2010 - 2026  Dirk Eddelbuettel and Romain Francois
 //
 // This file is part of Rcpp.
 //
@@ -80,7 +79,7 @@ namespace Rcpp {
             if (res == 0) {
                 return std::string("");
             } else {
-                res = ::snprintf(txtsec, 63, "%s.%06d", txtiso, m_us);
+                res = ::snprintf(txtsec, 63, "%s.%06u", txtiso, m_us);
                 if (res <= 0) {
                     return std::string("");
                 } else {

